cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -52,6 +52,7 @@
     "file": "document",
     "line": "line",
     "paint": "nonWhitespaceSequence",
+    "small paint": "nonWhitespaceOrQuoteSequence",
```

Comment:
As mentioned in Slack, I'd be tempted to go with

```suggestion
    "short paint": "nonWhitespaceOrQuoteSequence",
```

To continue the [`"short"` / `"long"`](http://redstartsystems.com/human-machine-grammar-the-rules#:~:text=the%20No%20button.-,Rule%2013,-Use%20phrase%20modes) terminology